server/handler/github_event_handler.go

```diff
@@ -0,0 +1,106 @@
+package handler
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/google/go-github/v45/github"
+	"github.com/mattermost/release-bot/config"
+	"github.com/mattermost/release-bot/log"
+	"github.com/mattermost/release-bot/model"
+	"github.com/mattermost/release-bot/store"
+	"github.com/palantir/go-githubapp/githubapp"
+	"github.com/pkg/errors"
+)
+
+const logEventType string = "github-event"
+
+type githubEventHandler struct {
+	Config            *config.Config
+	ClientCreator     githubapp.ClientCreator
+	GithubEvents      []string
+	EventContextStore store.EventContextStore
+}
+
+func (h *githubEventHandler) Handles() []string {
+	return h.GithubEvents
+}
+
+func (h *githubEventHandler) Handle(ctx context.Context, eventType, deliveryID string, payload []byte) error {
+	log.Info(logEventType).Msgf("%s is received!", eventType)
+	eventContext, err := model.ConvertPayloadToEventContext(eventType, payload)
+	if err != nil {
+		log.Info(logEventType).Msgf("Error occurred while deserializing request: %v", err)
+		return errors.Wrap(err, "failed to parse event payload")
+	}
+
+	eventContext.Log()
+
+	pipeline := model.GetTargetPipeline(eventContext, h.Config.Pipelines)
+
+	if pipeline != nil {
```

Comment:
let's make the opposite to avoid the `if/else`. I will be more readable `if pipeline == nil {} rest of the code`

Reply:
If condition is changed.